voxblox/include/voxblox/mesh/mesh.h

```diff
@@ -36,14 +37,25 @@ struct Mesh {
   typedef std::shared_ptr<Mesh> Ptr;
   typedef std::shared_ptr<const Mesh> ConstPtr;
 
+  static constexpr FloatingPoint kInvalidBlockSize = -1.0;
+
+  Mesh()
```

Comment:
If we combine all mesh-blocks into one global mesh, there is no point in using these block_size, origin and updated members.